content/administration/maintain/mailjet_api.rst

```diff
@@ -0,0 +1,221 @@
+============================
+Set up Mailjet API with Odoo
+============================
+
+Odoo is compatible with Mailjet's :abbr:`API (Application Programming Interface)` for mass mailing.
+Set up a dedicated mass mailing server through Mailjet by configuring settings in the Mailjet
+account and the Odoo database. In some circumstances, settings need to be configured on the custom
+domain's :abbr:`DNS (Domain Name System)` settings as well.
+
+Set up in Mailjet
+=================
+
+Create API credentials
+----------------------
+
+To get started, sign in to the `Mailjet Account Information <https://app.mailjet.com/account>`_
+page. Next, navigate to the :guilabel:`Senders & Domains` section and click on :guilabel:`SMTP and
+SEND API Settings`.
+
+.. image:: mailjet_api/api-settings.png
+   :align: center
+   :alt: SMTP and Send API Settings link in the Senders & Domains section of Mailjet.
+
+Then, copy the :abbr:`SMTP (Simple Mail Transfer Protocol)` configuration settings onto a notepad.
+They can be found under the :guilabel:`Configuration (SMTP only)` section. The :abbr:`SMTP (Simple
+Mail Transfer Protocol)` configuration settings include the server address, the security option
+needed (Use :abbr:`SSL (Secure Sockets Layer)`/:abbr:`TLS (Transport Layer Security)`), and the
+port number. The settings are needed to configured Mailjet in Odoo, which is covered in the
+:ref:`last section <maintain/mailjet-api/odoo-setup>`.
+
+.. note::
+   Here is `Mailjet's SMTP Configuration documentation <https://documentation.mailjet.com/hc/en-us/
+   articles/360043229473-How-can-I-configure-my-SMTP-parameters->`_ for more information.
```

Comment:
Same as here (please check all links and admonitions)
https://github.com/odoo/documentation/pull/3221#discussion_r1052038333

- If the tip is to suggest opening a link, then it's rather a seealso
- In RST, you can go to another line wherever you can write a space. So not in the middle of a link ^^
- Remove the region/language part of your URLs whenever possible (here the /en-us). This way, the link opens correctly for everyone, according to their cached settings on Microsoft's website or localization.

```suggestion
.. seealso::
   `Mailjet: How can I configure my SMTP parameters?
    <https://documentation.mailjet.com/hc/en-us/articles/360043229473>`_
```